llama.h

```diff
@@ -723,6 +723,15 @@ extern "C" {
                            float   p,
                           size_t   min_keep);
 
+    /// @details DYNATEMP! #TODO KALO
```

Comment:
TODO?

Reply:
I think the assumption he made is there'd be documentation / explanation in line with the other samplers

Reply:
Yes that is the assumption I made